modules/server_config/variables.tf

```diff
@@ -0,0 +1,587 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#--------#
+# Config #
+#--------#
+variable "composite_root_resources" {
+  description = "A list of root resources that Forseti will monitor. This supersedes the root_resource_id when set."
+  type        = list(string)
+  default     = []
+}
+
+variable "sendgrid_api_key" {
+  description = "Sendgrid.com API key to enable email notifications"
+  default     = ""
+}
+
+#----------------#
+# Forseti config #
+#----------------#
+variable "domain" {
+  description = "The domain associated with the GCP Organization ID"
+}
+
+variable "org_id" {
+  description = "GCP Organization ID that Forseti will have purview over"
+}
+
+variable "folder_id" {
+  description = "GCP Folder that the Forseti project will be deployed into"
+  default     = ""
+}
+
+
+variable "gsuite_admin_email" {
+  description = "G-Suite administrator email address to manage your Forseti installation"
+}
+
+variable "forseti_email_recipient" {
+  description = "Email address that receives Forseti notifications"
+  default     = ""
+}
+
+variable "forseti_email_sender" {
+  description = "Email address that sends the Forseti notifications"
+  default     = ""
+}
+
+#--------------------------#
+# Forseti config inventory #
+#--------------------------#
+variable "admin_max_calls" {
+  description = "Maximum calls that can be made to Admin API"
+  default     = "14"
+}
+
+variable "admin_period" {
+  description = "The period of max calls for the Admin API (in seconds)"
+  default     = "1.0"
+}
+
+variable "admin_disable_polling" {
```

Comment:
Could you change any `"true"` or `"false"` parameters in `variables.tf` and change them so they are `type = bool`?